src/app/pages/forgot-password/forgot-password.component.html

```diff
@@ -32,8 +32,33 @@ <h1 class="text-white">I Stand With Health Professionals.?</h1>
                 </div>
               </div>
               <div class="alert error-message">{{message}}</div>
```

Comment:
See how the previous, on master, the html looks like and revert it to that. On the forgot password all we need is only hte email address and you don't need to update this part.